### PR TITLE
Update networkperf test suite to support windows images

### PR DESCRIPTION
**Pull request by drewhli**
It's weird how the network performance test results don't reach anywhere close to the targets, especially in the non-Tier1 network tests (specifically for Windows)

### CODE REVIEW

**Comment by drewhli** [comment]
Seems like mostly a windows2016 issue.

File: imagetest/test_suites/networkperf/performance_test.go
Comment (koln67) [inline imagetest/test_suites/networkperf/performance_test.go:1]: TestEmpty should be removed

File: imagetest/test_suites/networkperf/performance_test.go [hunk 1]
```diff
@@ -19,45 +18,43 @@ func TestEmpty(t *testing.T) {
```
Comment (koln67) [inline imagetest/test_suites/networkperf/performance_test.go:18]: TestEmpty should be removed

Comment (drewhli) [reply imagetest/test_suites/networkperf/performance_test.go]: Done.

**Pull request merged**